### ISSUE TITLE
Incremental builds with Rollup exclude cached modules

### ISSUE THREAD

**Issue by portstrom** [tivac/modular-css#345]
When a CSS module is already in the cache passed as an option to Rollup, the CSS code from that module is never added to the CSS bundle. This happens for example when rollup-watch rebuilds after the initial build. It surprises me to see such a major problem, making modular-css-rollup completely unusable with rollup-watch, as I see evidence that much effort has gone into making modular-css-rollup work with rollup-watch.

Looking at the code of modular-css-rollup, I see that it creates a new `Processor` instance on every build, and only gives it CSS modules that are passed by Rollup to the `transform` function. Since Rollup does not call the `transform` function for modules that are in the cache and have not changed, cached modules are never given to the `Processor` instance.

I did a quick fix by added the following code to the start of the `ongenerate` function. It makes sure that all CSS modules are given to the `Processor` instance, even if they were never given to the `transform` function. The `Processor` class skips various tasks for modules that have already been added, so adding the same module twice seems to be safe.

```
bundle.bundle.modules.forEach(module => {
    let id = module.id
    if(filter(id) && id.slice(slice) === options.ext) {
        processor.string(id, module.originalCode)
    }
})
```

The drawbacks of this quick fix are that:

- All CSS modules are processed on every build, even if they have not changed since the previous build.
- The CSS modules are processed in a different order depending on which modules are in the cache.
- Because cached modules are processed after module transformation, the namer function must return stable names, as it is too late to export different names.

**Comment by tivac** [comment]
That must be new behavior, because the usage of a new `Processor` instance on each run was specifically added in https://github.com/tivac/modular-css/issues/158 to avoid staleness issues.

I'd love a PR if you can send one, otherwise I'll tackle this when I can!

**Comment by tivac** [comment]
This is fixed in `modular-css-rollup@6.0.2`.
